docs/content/administration/config-cheat-sheet.en-us.md

```diff
@@ -522,6 +522,10 @@ And the following unique queues:
   - `deletion`: User cannot delete their own account.
   - `manage_ssh_keys`: User cannot configure ssh keys.
   - `manage_gpg_keys`: User cannot configure gpg keys.
+- `EXTERNAL_USER_DISABLE_FEATURES`: **_empty_**: Disabled features ONLY if the user has an external login type (eg. LDAP, Oauth, etc.), could be `deletion`, `manage_ssh_keys`, `manage_gpg_keys` and more features can be added in future. This setting is independent from `USER_DISABLED_FEATURES` and supplements its behavior.
```

Comment:
Drop the "and more features can be added in future" part. Just explicitely list the valid values.

After these doc changes are done, copy the same sentence from Markdown into the ini imho. No need to maintain 2 different versions.